### PR TITLE
feat: local execution environment

### PR DESCRIPTION
**Pull request by PauloASilva**
Allow application to run locally withing an ephemeral docker container.
Container can be started running the following command from project's
root directory:

```
$ docker-compose up -d
```

Source code can be edited in the host machine. Changes are instantly
available.

To stop, run:

```
$ docker-compose down
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,23 @@ Our electricity is not made equal and can be associated with higher or lower car
 
 The web app and the underlying data pipeline was built using open source tools. Web app's front-end was built using Shiny, electricity composition was extracted from National Grid's [Carbon Intensity API](https://carbonintensity.org.uk/) using [{intensegRid}](https://kkulma.github.io/intensegRid/articles/intro-to-carbon-intensity.html) package. Data and deployment pipelines were developed using GitHub Actions: `schedule.yml` pulls the carbon intensity data from the API, saves it in `data/daily_ci.rds` file daily and pushes the changes to the repo; `deploy.yml` deploys the changes to the [shinyapps.io server](https://kasiakulma.shinyapps.io/carbon-intensity-app/) on push. 
 
+## Getting start
```
Comment (KKulma) [inline README.md]: could you please correct the typo (to Getting start**ed**)?

**Review by KKulma — CHANGES_REQUESTED** [review]
thank for that - very neat PR! Could you just correct a small typo and rebase to `main` before I approve and merge it in? Big thanks!

File: README.md (hunk 1 above)
Comment (PauloASilva) [reply README.md]: PR was updated accordingly.

**Review by KKulma — APPROVED** [review]
awesome, thanks again for your contributions!

**Pull request merged**